api/v1/paths/notes.js

```diff
@@ -1,9 +1,10 @@
 const appRoot = require('app-root-path');
 
-const { errorHandler } = appRoot.require('errors/errors');
-const { openapi: { paths } } = appRoot.require('utils/load-openapi');
 const notesDAO = require('../db/json/notes-dao');
 
+const { paths } = appRoot.require('app').locals.openapi;
```

Comment:
Another style question. Which one should we use?

```js
const { paths } = appRoot.require('app').locals.openapi;
```

or

```js
const { local: { openapi: { paths } } } = appRoot.require('app');
```

Personally, I think the style you used is cleaner and easier to read. I'm just not sure is there any suggested practice or not. What do you think?

Reply:
This style question was added to [CO-1366](https://jira.sig.oregonstate.edu/browse/CO-1366)